source/state_representation/README.md

```diff
@@ -1,18 +1,21 @@
 # state_representation
 
-This library provides a set of classes to represent **states** in **cartesian** or **joint** spaces, **parameters**, or **geometrical shapes** that can be used as obstacles.
-Those are a set of helpers functions to handle common concepts in robotics such as transformations between frames and the link between them and the robot state.
+This library provides a set of classes to represent **states** in **cartesian** or **joint** spaces, **parameters**, 
```

Comment:
What happened here with all those new lines?

Reply:
Looks like explicit line breaking, probably by the formatter (max line length). It will render in the markdown the same way, but I generally prefer it this way where the source lines are broken up. It's much more readable especially when the IDE doesn't wrap lines in the source by default

Reply:
It was actually not automatic because auto formatting messes up stuff in the code block but indeed I used the line marker on clion and broke there